mk_mingw.mak

```diff
@@ -16,8 +16,8 @@ VPATH = . ./main ./parsers
 ctags.exe: OPT = -O4 -Os -fexpensive-optimizations
 ctags.exe: LDFLAGS = -s
 dctags.exe: OPT = -g
-dctags.exe: DEBUG = -DDEBUG
-dctags.exe: SOURCES += debug.c
+dctags.exe: CFLAGS += -DDEBUG
+# dctags.exe: SOURCES += $(MAIN_DIR)/debug.c
```

Comment:
in source.mak line 106, debug.c already included, 
so it is unecessary to put it in mk_mingw.mak (the compiler will complaint anyway)

DEBUG_SOURCES = $(MAIN_DIR)/debug.c

SOURCES = $(MAIN_SOURCES) $(PARSER_SOURCES) $(DEBUG_SOURCES)